rmc/pipeline/regional_constraint.py

```diff
@@ -395,13 +405,13 @@ def main(args):
                 p_value = args.p_value
             rmc_ht = rmc_ht.annotate_globals(p_value=p_value)
 
-            logger.info("Writing out RMC results...")
-            rmc_ht.write(
-                rmc_results.versions[args.freeze].path, overwrite=args.overwrite
-            )
+            # logger.info("Writing out RMC results...")
+            # rmc_ht.write(
```

Comment:
Pylint is failing due to couple unused imports here, do we just comment them out for now?

Reply:
we only commented this section out to avoid overwriting this HT -- we should remove the comments before merging (we don't want to merge this code with the comments)

Reply:
Done (uncommenting part)